### ISSUE TITLE
WOTLK Classic PTR Unknown event CURSOR_UPDATE

### ISSUE THREAD

**Issue by Jakobud** [AdiAddons/AdiBags#654]
**Describe the bug**
```
3x AdiBags\core\EventHandlers.lua:44: Attempt to register unknown event "CURSOR_UPDATE"
[string "=[C]"]: in function `RegisterEvent'
[string "@AdiBags\core\EventHandlers.lua"]:44: in function `OnUsed'
[string "@AdiBags\libs\CallbackHandler-1.0\CallbackHandler-1.0-7.lua"]:76: in function <...ags\libs\CallbackHandler-1.0\CallbackHandler-1.0.lua:59>
[string "=(tail call)"]: ?

Locals:
(*temporary) = <unnamed> {
 0 = <userdata>
}
(*temporary) = "CURSOR_UPDATE"
```

**To Reproduce**
1. Load the addon

**World of Warcraft Release and Version (Retail 9.2.7, etc)**
- WOTLK Classic PTR Interface # 30400

**Have you tried disabling all your other addons to ensure only this addon causes the issue?**
 - Yes

I'm pretty sure that CURSOR_UPDATE is not a removed event so I'm not what is going on here. Whatever it is, it's a showstopper for Adibags (the best bag addon period).

**Comment by Cidan** [comment]
Ah, just triggered it. Thanks for the report!

**Comment by Cidan** [comment]
Update: It looks like this is broken in quite a few places, including Ace. Extracting the Blizzard interface files yielded no reference, but this will break due to Ace anyway. Going to hold off on this until pre-patch launch and see if Blizzard pushes any fixes.

**Comment by Cidan** [comment]
This has been fixed, pushed to master, and released as v1.9.30.

**Comment by Cidan** [comment]
Yes -- we'll be updating curseforge at some point today!
